### PR TITLE
Added FAQ to Google API Key

### PR DESCRIPTION
**Pull request by ftmzhrsckldr**

### CODE REVIEW

File: docs/account/my-organization/api-integrations/adding-google-play-service-account.md [hunk 1]
```diff
@@ -71,4 +65,23 @@ Make sure this account has access to **Releases**, **Store presence**, and **App
 
 Then click **Invite User**. Your account key is ready. 🎉
 
-To add a key, go to [My Organization](/account/my-organization) and press the "Add New" button (or the "Manage" button first if you have saved keys) next to the "Google Play Developer API Keys" item under the Connections section.
+14. To add the key on Appcircle, follow these steps:
+
+    a. Navigate to [My Organization](/account/my-organization)n.
```
Comment (boztopuz) [inline docs/account/my-organization/api-integrations/adding-google-play-service-account.md]:
```suggestion
    a. Navigate to [My Organization](/account/my-organization).
```

**Review by boztopuz — APPROVED** [review]

File: docs/account/my-organization/api-integrations/adding-google-play-service-account.md (hunk 1 above)
Comment (boztopuz) [reply docs/account/my-organization/api-integrations/adding-google-play-service-account.md]: @ftmzhrsckldr I fixed this typo. Thnks for your effort. I approved

File: docs/account/my-organization/api-integrations/adding-google-play-service-account.md [hunk 2]
```diff
@@ -71,4 +65,23 @@ Make sure this account has access to **Releases**, **Store presence**, and **App
...
+    c. Click the `Manage` button if you have saved keys, or directly click the `Add New` button.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/google-service14.png' />
+
+
+## FAQ
+
+**Can I retrieve the JSON private key that I uploaded to Appcircle?**
+
+No, for security reasons, the JSON file you uploaded cannot be downloaded again from Appcircle.
```
Comment (erenkan) [inline docs/account/my-organization/api-integrations/adding-google-play-service-account.md:83]:
```suggestion
No, for security reasons, you cannot download the JSON file you uploaded to Appcircle again.
```

File: docs/account/my-organization/api-integrations/adding-google-play-service-account.md [hunk 3]
```diff
@@ -71,4 +65,23 @@ Make sure this account has access to **Releases**, **Store presence**, and **App
...
+
+**After uploading the JSON, why can't I save it?**
```
Comment (erenkan) [inline docs/account/my-organization/api-integrations/adding-google-play-service-account.md:85]:
```suggestion
**Why can’t I save the JSON after uploading it?**
```

File: docs/account/my-organization/api-integrations/adding-google-play-service-account.md [hunk 4]
```diff
@@ -71,4 +65,23 @@ Make sure this account has access to **Releases**, **Store presence**, and **App
...
+
+You may have skipped the first step. Please ensure you have enabled the Service account.
```
Comment (erenkan) [inline docs/account/my-organization/api-integrations/adding-google-play-service-account.md:87]:
```suggestion
You might have missed the first step. Please ensure that you have enabled the Service Account.
```

**Review by erenkan — CHANGES_REQUESTED** [review]

**Review by erenkan — APPROVED** [review]

**Pull request merged**